front-end/src/Login.js

```diff
@@ -0,0 +1,35 @@
+import './Login.css';
+import React, { Component } from "react";
+import { Link } from 'react-router-dom';
+
+
+
+    function handleChange(e) {
+        this.setState({[e.target.name]: e.target.value })
```

Comment:
Residual class-based component state management should be removed or replaced with hooks instead (i.e. `const [state, setState] = useState('');`). As this is, every occurrence of `this.state` breaks the code and gives errors.